app/src/components/SubregionMap.vue

```diff
@@ -28,6 +28,7 @@ export default {
                 self.allData = json.features;
                 self.nationalFeature = json.features.slice(52,53);
                 self.subregionData = json.features.slice(53,json.features.length);
+                console.log(self.nationalFeature);
```

Comment:
You should remove this debug print

Reply:
Thanks @j-tafoya . I made both changes (updated the SubregionMap.vue to remove the debug print and removed the *.dbf and*.shp files). Let me know if you have additional comments, thx!